wiki/Daily_challenge/en.md

```diff
@@ -0,0 +1,40 @@
+# Daily Challenge
+
+*Daily challenge* is a mode where the main task is to pass a selected beatmap that changes everyday sometimes with specific [mods](wiki/Gameplay/Game_modifier).
```

Comment:
```suggestion
The **daily challenge** is a multiplayer mode in osu!(lazer) where the main task is to pass a selected beatmap that changes everyday sometimes with specific [mods](wiki/Gameplay/Game_modifier).
```
For now, *multiplayer* is a bit stretched as a term. The devs planned to add more features later on though, with for example collaboratively achieving a certain total score.